repository-commons/src/main/java/io/aiven/elasticsearch/repositories/io/CryptoIOProvider.java

```diff
@@ -30,7 +30,7 @@
 
 import com.github.luben.zstd.ZstdInputStream;
 import com.github.luben.zstd.ZstdOutputStream;
-import org.elasticsearch.common.io.Streams;
+import org.elasticsearch.core.internal.io.Streams;
```

Comment:
I'm a bit worried by a module name 'internal'. What do you think?

Reply:
Well same for me. Let's keep it so far this way and excluded in the future.